src/org/jgroups/protocols/TP.java

```diff
@@ -207,6 +207,9 @@ public String getBundlerClass() {
     public long             getTimeServiceInterval() {return time_service_interval;}
     public <T extends TP> T setTimeServiceInterval(long t) {this.time_service_interval=t; return (T)this;}
 
+    public boolean          useVThreads()          {return use_vthreads;}
```

Comment:
That's pretty bad naming. Methods returning a boolean should have names that start with "is" or "has".

Reply:
I tried to stay consistent with the version from <=5.3 which useVirtualThreads(), or other existing getters i.e. receiveOnAllInterfaces().
Should I still prefix with "is" for this?

Reply:
The naming in this class is already very inconsistent, e.g. isTrace is both getter and setter but by convention these should be setTrace/isTrace, so looking at past code for inspiration is not great.

Replying on parametric polymorphism for getter/setter is an antipattern.

Reply:
Also, the method name doesn't have to reflect the internal variable this sets (especially since the variable naming conventions in this particular projects are not Java-based..)